pinot-broker/src/main/java/org/apache/pinot/broker/routing/instanceselector/BaseInstanceSelector.java

```diff
@@ -291,5 +296,6 @@ public SelectionResult select(BrokerRequest brokerRequest, List<String> segments
    * ONLINE/CONSUMING instances). If enabled instances are not {@code null}, they are sorted in alphabetical order.
    */
   abstract Map<String, String> select(List<String> segments, int requestId,
-      Map<String, List<String>> segmentToEnabledInstancesMap, Map<String, String> queryOptions);
+      Map<String, List<String>> segmentToEnabledInstancesMap, Map<String, String> queryOptions,
```

Comment:
No need to change this. We should make `_adaptiveServerSelector` protected so that it can be accessed by the subclasses

Reply:
Good point. Done.